src/main/java/com/javadiscord/javabot/other/Database.java

```diff
@@ -24,65 +23,47 @@
 
     private static final org.slf4j.Logger logger = LoggerFactory.getLogger(Database.class);
 
-    private static final String[] databases = new String[]{"userdata", "other"};
-    private static final String[] userdataCollections = new String[]{"potential_bot_list", "users", "warns"};
-    private static final String[] otherCollections = new String[]{"config", "customcommands", "expert_questions", "open_submissions", "reactionroles", "starboard_messages", "submission_messages"};
+    private static final Map<String, String[]> DB_COLS = new HashMap<>();
```

Comment:
It would be nice to have either
- a more descriptive name of what this variable is for, or...
- javadoc description of this variable.

Reply:
Almost all methods need a javadoc.